app/views/components/fields/LabelledValue.scala

```diff
@@ -16,8 +16,11 @@
 
 package views.components.fields
 
+import play.api.libs.json.JsValue
+
 case class LabelledValue(label: String, value: String)
 
 object LabelledValue {
   def apply(labelAndValue: String): LabelledValue = LabelledValue(labelAndValue, labelAndValue)
+  def apply(labelAndValue: JsValue): LabelledValue = LabelledValue(labelAndValue.toString, labelAndValue.toString)
```

Comment:
Yeah, it's better now. I'm just wondering why we need to have the same `label` and `value`?

Of course you can create now case class with two arguments, so it's okay, but I'm just curious what case is covered by it.

Reply:
Two apply is good enough - you have still strongly types and everything should work perfectly.

Reply:
@PatrykRudnicki this depends on how MultipleItemsHelper  works - it compare sent value with values in model